test-browser/helpers/contracts.js

```diff
@@ -162,6 +163,9 @@ function addInstance (browser, address, callback) {
     browser.click('div[class^="atAddress"]')
       .perform((client, done) => {
         browser.execute(function () {
+          if (!ethJSUtil.isValidChecksumAddress(address)) {
```

Comment:
I think it is better to add params in `addInstance` to describe what you expect.
like `function addInstance (browser, address, isValidFormat, isValidChecksum, callback) { ...`
and test if the ui behaves accordingly

Reply:
@yann300 -- Sure. If you can let me know the exact requirements for these tests, I'd be able to handle those use cases accordingly by adding those `params`.

Reply:
are yo still working on that @eswarasai? I think everything is fine, you just need to update the above test.

Reply:
@yann300 -- Yes. Sorry about the delay. I'll do that and update within the next 12 hours. Thanks

Reply:
@yann300 -- I've pushed the changes as per requested above. I've added 2 more params : `isValidFormat, isValidChecksum` to `addInstance`. I haven't updated the tests, they're as it is for now. Let me know if there's a test needed for invalid address as well. Thanks.